packages/manager/src/features/CloudPulse/Widget/components/CloudPulseIntervalSelect.tsx

```diff
@@ -118,36 +122,42 @@ export const CloudPulseIntervalSelect = React.memo(
       defaultValue
     );
 
+    const theme = useTheme();
+
     return (
-      <StyledWidgetAutocomplete
-        isOptionEqualToValue={(
-          option: IntervalOptions,
-          value: IntervalOptions
-        ) => {
-          return option?.value === value?.value && option?.unit === value?.unit;
-        }}
-        onChange={(
-          _: React.SyntheticEvent,
-          selectedInterval: IntervalOptions
-        ) => {
-          setSelectedInterval(selectedInterval);
-          onIntervalChange({
-            unit: selectedInterval?.unit,
-            value: selectedInterval?.value,
-          });
-        }}
-        textFieldProps={{
-          hideLabel: true,
-        }}
-        autoHighlight
-        disableClearable
-        fullWidth={false}
-        label="Select an Interval"
-        noMarginTop={true}
-        options={[autoIntervalOption, ...availableIntervalOptions]}
-        sx={{ width: { xs: '100%' } }}
-        value={selectedInterval}
-      />
+      <CloudPulseTooltip title={'Data aggregation interval'}>
+        <Autocomplete
+          isOptionEqualToValue={(
+            option: IntervalOptions,
+            value: IntervalOptions
+          ) => {
+            return (
+              option?.value === value?.value && option?.unit === value?.unit
+            );
+          }}
+          onChange={(
+            _: React.SyntheticEvent,
+            selectedInterval: IntervalOptions
+          ) => {
+            setSelectedInterval(selectedInterval);
+            onIntervalChange({
+              unit: selectedInterval?.unit,
+              value: selectedInterval?.value,
+            });
+          }}
+          textFieldProps={{
+            hideLabel: true,
+          }}
+          autoHighlight
+          disableClearable
+          fullWidth={false}
+          label="Select an Interval"
+          noMarginTop={true}
+          options={[autoIntervalOption, ...availableIntervalOptions]}
+          sx={getAutocompleteWidgetStyles(theme)}
```

Comment:
Same thing here. `sx` can take a function. This will allow us to remove the `useTheme` call above 
```suggestion
          sx={getAutocompleteWidgetStyles}
```